src/utils/helpers/structure/updateSectionsProgress.ts

```diff
@@ -19,6 +19,19 @@ interface GetSectionsProgressProps {
   }) => void
 }
 
+/**
+ * @function updateSectionsProgress
+ * Update sections strucutre and add the progress on each section.
```

Comment:
```suggestion
 * Update sections structure and add the progress on each section.
 ```
Ha ha, same thing again ;) We all have those words we constantly mis-type. For me it's "seperated" (should be "separated") 😆